examples/DotNetCore/WebDemoNet6/WebDemoNet6/Program.cs

```diff
@@ -0,0 +1,55 @@
+using Microsoft.FeatureManagement;
+using WebDemoNet6;
+
+var builder = WebApplication.CreateBuilder(args);
+
+// Load configuration from Azure App Configuration
+builder.Configuration.AddAzureAppConfiguration(options =>
+{
+    options.Connect(Environment.GetEnvironmentVariable("ConnectionString"))
+           // Load all keys that start with `WebDemo:` and have no label
+           .Select("WebDemo:*")
+           // Configure to reload configuration if the registered key 'WebDemo:Sentinel' is modified.
+           // Use the default cache expiration of 30 seconds. It can be overriden via AzureAppConfigurationOptions.SetCacheExpiration.
```

Comment:
```suggestion
           // Use the default cache expiration of 30 seconds. It can be overriden via AzureAppConfigurationRefreshOptions.SetCacheExpiration.
```

Reply:
Good catch. Fixed it in this and a few other projects. Thanks!